site/reference/fn/eval/README.md

```diff
@@ -79,6 +79,8 @@ fn-args:
 
 --image:
   Container image of the function to execute e.g. `gcr.io/kpt-fn/set-namespace:v0.1`.
+  For convenience, if full image path is not specified, `gcr.io/kpt-fn/` is added as default prefix.
```

Comment:
What about reference doc for `render`? How about `fn doc` command?

Reply:
Made changes to render and fn doc as well.